cpp/source/geometry.templates.h

```diff
@@ -36,13 +36,13 @@ std::tuple<std::pair<dealii::Point<2>, dealii::Point<2>>,
 foo<2>(std::shared_ptr<boost::property_tree::ptree const> const &database)
 {
   // clang-format off
-  double const anode_collector_width   = database->get<double>("anode_collector_width");
-  double const anode_electrode_width   = database->get<double>("anode_electrode_width");
-  double const separator_width         = database->get<double>("separator_width");
-  double const cathode_electrode_width = database->get<double>("cathode_electrode_width");
-  double const cathode_collector_width = database->get<double>("cathode_collector_width");
-  double const sandwich_height         = database->get<double>("sandwich_height");
-  double const tab_height              = database->get<double>("tab_height");
+  double const anode_collector_width   = 0.01 * database->get<double>("anode_collector_thickness");
+  double const anode_electrode_width   = 0.01 * database->get<double>("anode_electrode_thickness");
+  double const separator_width         = 0.01 * database->get<double>("separator_thickness");
+  double const cathode_electrode_width = 0.01 * database->get<double>("cathode_electrode_thickness");
+  double const cathode_collector_width = 0.01 * database->get<double>("cathode_collector_thickness");
+  double const sandwich_height         = 0.0001 * database->get<double>("geometric_area");
+  double const tab_height              = 0.01 * database->get<double>("tab_height");
```

Comment:
Can you do the same here. Also I think that we should move the definition of ```to_meters``` to another file. So if one day we need to change to feet, there is only place in the code to change.